changelog/21.0/21.0.0/summary.md

```diff
@@ -19,12 +19,13 @@
     - **[Reference Table Materialization](#reference-table-materialization)**
     - **[New VEXPLAIN Modes: TRACE and KEYS](#new-vexplain-modes)**
     - **[Errant GTID Detection on Vttablets](#errant-gtid-vttablet)**
+    - **[Automatically Replace MySQL auto_increment Clauses with Vitess Sequences](#auto-replace-mysql-autoinc-with-seq)**
 
-## <a id="major-changes"/>Major Changes
+## <a id="major-changes"/>Major Changes</a>
```

Comment:
Was adding `</a>` to every line done by your editor? Seems unnecessary.

Reply:
I did it as w/o it the content is messed up in some readers (the preview one I use). It’s proper to end it.